src/sklearn_evaluation/plot/classification.py

```diff
@@ -141,7 +144,7 @@ def _confusion_matrix(y_true, y_pred, normalize):
     return cm
 
 
-@SKLearnEvaluationLogger.log(feature="plot")
+@tel_cm.log_call(action="function")
```

Comment:
I see here the action is function, super generic, how would we know it was a confusion matrix?

Reply:
I assume by the group? But still why call it function vs it's name